teamsalerts/main.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_lambda_function" "sns_to_teams" {
+  filename         = "${path.module}/lambda/dist/archive.zip"
+  source_code_hash = filebase64sha256("${path.module}/lambda/dist/archive.zip")
+  function_name    = var.name
+  handler          = "lambda.handler"
+  role             = aws_iam_role.lambda.arn
+  runtime          = "nodejs12.x"
```

Comment:
This is the only thing that stood out to me - it looks like AWS is deprecating node 12 as of March 31st https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html Would it be reasonable to switch to a newer version?

Reply:
so ok I can try upgrading but I _think_ because DS-infra uses the 2.70 provider, we actually can't use a newer runtime

Reply:
ok yeah so I'm getting this error when I switch to node 14:
```
λ ~/src/lcm/DS-Infrastructure/terraform/environments/test/ develop* aws-vault exec massgov-core -- terraform plan
Acquiring state lock. This may take a few moments...

│ Error: expected runtime to be one of [dotnetcore1.0 dotnetcore2.0 dotnetcore2.1 dotnetcore3.1 go1.x java8 java11 nodejs4.3 nodejs4.3-edge nodejs6.10 nodejs8.10 nodejs10.x nodejs12.x provided python2.7 python3.6 python3.7 python3.8 ruby2.5 ruby2.7], got nodejs14.x
│
│   with module.test_notifications.module.sns_to_teams.aws_lambda_function.default,
│   on .terraform/modules/test_notifications.sns_to_teams/lambda/main.tf line 1, in resource "aws_lambda_function" "default":
│    1: resource "aws_lambda_function" "default" {
│
╵
```

bumping the aws provider to 4.56.0 squashes the error, but this isn't the version that's used in ds-infrastructure (something I also found prevents us from using the latest version of the lambda module). can we bump the provider version? I'm guessing there's good reason it's so much lower than the latest.

Reply:
Gonna resolve this convo as this version will get bumped when we do this ticket: https://massgov.atlassian.net/browse/DP-27425